application/decoder/cdl.py

```
@@ -0,0 +1,58 @@
+    def parseLogLine(self, line):
+        '''
+            Parse the log line and save the relevant data.
+        '''
+        delimiter = line[0]
+
+        if delimiter == CDL.LINE_TYPE_DELIMITER["IR_HEADER"]:
+            self.header = json.loads(line)
+        elif delimiter == CDL.LINE_TYPE_DELIMITER["EXCEPTION"]:
+            self.exception = line
+        elif delimiter == CDL.LINE_TYPE_DELIMITER["VARIABLE"]:
+            self.exception = line
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)